### ISSUE TITLE
What is the difference between 'head' and 'intermediate' in 'obj_embedding_head'?

### ISSUE THREAD

**Issue by Cohesion97** [amirbar/DETReg#50]
https://github.com/amirbar/DETReg/blob/0a258d879d8981b27ab032b83defc6dfcbf07d35/models/backbone.py#L156-L177

It seems 'head' is the new training setting that uses dim=128 to align features. But dim=512 ('intermediate') is used in the paper. Does it mean that we should change to dim=128 ('head') to achieve better performance of DETReg?

Thanks.

**Comment by amirbar** [comment]
Hi, we've explored both options. Holding everything else fixed, both options perform similarly well.
